### DISCUSSION TITLE
./docker-entrypoint.sh: line 82: ./fw-rules.sh: Is a directory

### DISCUSSION THREAD

**Discussion by macollybarbosa** [d3vilh/openvpn-ui#37] [Q&A]
Hello, I would like to ask for help because I have a problem with "openvpn-server", I can get it to work, but when I restart the server when I return it shows the log below.

**Container logs**
```
net.ipv4.ip_forward = 1
Configuring iptables...
NAT for OpenVPN clients
Blocking ICMP for external clients
Blocking internal home subnet to access from external openvpn clients (Internet still available)
Applying firewall rules
./docker-entrypoint.sh: line 82: ./fw-rules.sh: Is a directory
```

To make it work again, I need to delete everything and rebuild, this way it works again, but I need to create all the certificates again, but if I restart the docker container it shows the error and no longer starts, could you help me?

**Comment by reamasesa** [comment]
> Hello, I would like to ask for help because I have a problem with "openvpn-server", I can get it to work, but when I restart the server when I return it shows the log below.
> 
> **Container logs**
> 
> ```
> net.ipv4.ip_forward = 1
> Configuring iptables...
> NAT for OpenVPN clients
> Blocking ICMP for external clients
> Blocking internal home subnet to access from external openvpn clients (Internet still available)
> Applying firewall rules
> ./docker-entrypoint.sh: line 82: ./fw-rules.sh: Is a directory
> ```
> 
> To make it work again, I need to delete everything and rebuild, this way it works again, but I need to create all the certificates again, but if I restart the docker container it shows the error and no longer starts, could you help me?

Hi,

I believe it is related to how you define the "volume" section in the docker-compose.yml (I had this issue a few times), you have to create an empty file for the container to fire up properly

How do you use this? Docker compose? Docker file? Share your setup file, it would help

**Comment by bobdvb** [comment]
I had the same issue as this, after using the Docker Compose configuration.

I had to:
* rmdir fw-rules.sh
* touch fw-rules.sh

Then the script continues.
